tests/page_test.go

```diff
@@ -33,6 +35,51 @@ type jsFrameBaseOpts struct {
 
 const sampleHTML = `<div><b>Test</b><ol><li><i>One</i></li></ol></div>`
 
+func TestNestedFrames(t *testing.T) {
+	t.Parallel()
+
+	buf := bytes.NewBufferString("")
+	tb := newTestBrowser(t,
+		withFileServer(),
+		func(tb *testBrowser) {
```

Comment:
Can we check the effect of clicking the button without using logs? For example, [like this](https://github.com/grafana/xk6-browser/blob/7ca6bde4ce483b4c3228e09fa6cbcf0f1d03d0d1/tests/locator_test.go#L56-L63).

Reply:
done